docs/kata-containers.md

```diff
@@ -0,0 +1,49 @@
+# Kata containers support in CRI-RM
+
+Kata containers represent an alternative runtime to Runc containers,
+and are conceptually defined by a containers which replace their sandbox 
+with a lightweight Virtual Machine.
+
+In order to enable Kata container in a Kubernetes-CRI-RM stack, both Kubernetes 
+and the Container Runtime Engine need to be aware of the new runtime environment:
+
+  * The Container Runtime Engine can only be CRI-o or containerd, and will need to
+   have the runtimes enabled in their configuration files.
+  * Kubernetes will also have to be made aware via a "RuntimeClass"  resource
+
+After these prerequesites are satisfied, the configuration file for the target  Kata Container, will have to have the flag "SandboxCgroupOnly" to true. As of Kata 2.0 this is the only way Kata containers can with the Kubernetes cgroup naming conventions.
+
+   ```toml
+   ...
+   # if enabled, the runtime will add all the kata processes inside one dedicated cgroup.
+   # The container cgroups in the host are not created, just one single cgroup per sandbox.
+   # The runtime caller is free to restrict or collect cgroup stats of the overall Kata sandbox.
+   # The sandbox cgroup path is the parent cgroup of a container with the PodSandbox annotation.
+   # The sandbox cgroup is constrained if there is no container type annotation.
+   # See: https://godoc.org/github.com/kata-containers/runtime/virtcontainers#ContainerType
+   sandbox_cgroup_only=true
+   ...
+   ``` 
+
+Controllers known to be compatible with Kata and CRI-RM:
+   * cri & cgroup controller
+   * RDT controller
```

Comment:
I'd remove this. Commit 85c8d9507894f621e8e635c9caaf3ae0cff490f1 should have fixed the `page-migrate` controller to work s well.